extensions/BugModal/template/en/default/bug_modal/new_comment.html.tmpl

```diff
@@ -13,14 +13,14 @@
 
 <div id="add-comment">
 
-  [% IF !bug.check_can_change_field('longdesc', 0, 1) %]
+  [% IF bug && !bug.check_can_change_field('longdesc', 0, 1) %]
     <div id="new-comment-notice">
       You are not allowed to make an additional comment on this [% terms.bug %].
     </div>
     [% RETURN %]
   [% END %]
 
-  [% IF user.is_insider %]
+  [% IF bug && user.is_insider %]
     <div id="add-comment-private"
       title="Make comment visible only to members of the '[% Param('insidergroup') FILTER html %]' group"
     >
```

Comment:
There are a few more items around the comment that should be hidden:
![remove these options](http://imgur.com/uChiEgs.png)